src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item/ProteinsListItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import ProteinsListItemInfo from '../proteins-list-item-info/ProteinsListItemInfo';
+
+import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
+
+import { getCommaSeparatedNumber } from 'src/shared/helpers/formatters/numberFormatter';
+import { getFeatureCoordinates } from 'src/content/app/entity-viewer/shared/helpers/entity-helpers';
+
+import transcriptsListStyles from 'src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss';
+import styles from './ProteinsListItem.scss';
+
+type Props = {
+  transcript: Transcript;
+};
+
+const ProteinsListItem = (props: Props) => {
+  const { transcript } = props;
+
+  const [shouldShowInfo, setShouldShowInfo] = useState(false);
+  const toggleListItemInfo = () => setShouldShowInfo(!shouldShowInfo);
+  const getSplicedRNALength = () => {
+    const rnaLength = transcript.exons.reduce((length, exon) => {
+      const { start, end } = getFeatureCoordinates(exon);
+      return length + (end - start + 1);
+    }, 0);
+
+    return getCommaSeparatedNumber(rnaLength);
+  };
+
+  const midStyles = classNames(transcriptsListStyles.middle, styles.middle);
+
+  return (
+    <div className={styles.proteinsListItem}>
```

Comment:
No such thing in your styles, apparently

![image](https://user-images.githubusercontent.com/6834224/81939224-f812f080-95ed-11ea-9880-2b786a6bf3fb.png)

Reply:
✔️